### PR TITLE
#2100: added manufacturer test data

### LINKED ISSUE #2100: Create manufacturer test data
### Desired Changes
We need more seed data that we can use to test our endpoints. This ticket is to create a new Manufacturer in the test data.

### Acceptance Criteria
- There is a new manufacturer in the test data

### Proposed Solution
- Take a look at the schema for a manufacturer and take a look at the required fields
- Look at how our other data is defined

### Screenshots (as needed)

_No response_

### PR DESCRIPTION
**Pull request by lilyshiomitsu**
## Changes

added new manufacturer test data to projects.test-data.ts

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #2100

### CODE REVIEW

**Review by Aaryan1203 — APPROVED** [review]
Great work on this!

File: src/backend/tests/test-data/projects.test-data.ts [hunk 1]
```diff
@@ -244,6 +244,14 @@ export const manufacturer2: Manufacturer = {
   materials: [material1]
 };
 
+export const manufacturer3: Manufacturer = {
+  name: 'Manufacturer3',
```
Comment (RChandler234) [inline src/backend/tests/test-data/projects.test-data.ts:248]: can you change the name to be an actual manufacturer (of your choosing)

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/backend/tests/test-data/projects.test-data.ts (hunk 1 above)
Comment (lilyshiomitsu) [reply src/backend/tests/test-data/projects.test-data.ts:248]: The two test data right above the one I added also have Manufacturer1 and Manufacturer2 for the names, should I change those as well

Comment (lilyshiomitsu) [reply src/backend/tests/test-data/projects.test-data.ts]: done

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**